auslib/web/common/swagger/definitions.yml

```diff
@@ -324,6 +324,11 @@ definitions:
         type: ["boolean", "null"]
         example: true
 
+      jaws:
+        description: Whether or not the Rule should apply to queries that indicate an incompatible version of the JAWS screen reader. If set to false or true, the query and the rule must match precisely. If set to null, the jaws value in the updateQuery is ignored.
```

Comment:
Wow! Up until this point, I though JAWS was some Java-related capability. Thanks for documenting that. TIL JAWS!